### PR TITLE
Create Code of Conduct

### PR DESCRIPTION
**Pull request by alexroseb**
Adapted from GitHub's proposed Code of Conduct to add some language especially suited to this project.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at `EMAIL ADDRESS`. All
```
Comment (alexroseb) [inline CODE_OF_CONDUCT.md]: @scottaohara What email address would fit best here?

**Comment by alexroseb** [comment]
@ericwbailey Definitely agree! I added specific language about not using ableist words (with a link to a page explaining that)

**Comment by alexroseb** [comment]
Mostly I just believe we should have a CoC and wanted to start a conversation around that.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (scottaohara) [inline CODE_OF_CONDUCT.md]: we have hello@a11yproject.com on the website.  

That's the best one to use as far as I know.

**Comment by ericwbailey** [comment]
I've spoken to some friends active in the community, and they've identified a few good CoCs to reference:

- https://2018.cssconf.com.au/code-of-conduct
- https://www.ubuntu.com/community/code-of-conduct
- https://austindesignweek.org/code-of-conduct
- https://2018.xoxofest.com/conduct

There's also a quote from her about setting the CoC's tone that really resonated with me:
> Describe the high standard that you would like the atmosphere to rise to.

I'm also going to make some inquiries to some organizations and see if we can work something out in terms of a professional audit.

**Comment by ericwbailey** [comment]
I'm going to merge this as a base to work from. Thank you for submitting it, @alexroseb! It's very much needed and going to make for a strong foundation to build off of 💪

**Pull request merged**